### DISCUSSION TITLE
Why SQLITE instead of H2?

### DISCUSSION THREAD

**Discussion by Nucceteere** [TagStudioDev/TagStudio#548] [Ideas]
As you know one of the goals of this project is to be able to handle big data like a terabyte or two. But SQLite is slower in that aspect compared to H2 which is much faster when handling big data. Wouldn't it be better to use H2 to support the users with files over 500GB?

**Comment by CyanVoxel** [comment]
Files in TagStudio are not stored within the library database, but rather just the file paths themselves as well as tags and other metadata. Also, H2 appears to be written in Java and requires that as a dependency? It doesn't seem like a good fit for the project compared to SQLite.

**Reply by Nucceteere** [reply]
Got it, thanks.
